lib/request.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const http = require('https');
+
+const post = (url, data, cb) => {
+  const options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Content-Length': Buffer.byteLength(data),
+    },
+  };
+
+  let errored = false;
+
+  const req = http.request(url, options, (res) => {
+    res.setEncoding('utf8');
+
+    const { statusCode } = res;
+
+    let body = '';
+    res.on('data', (chunk) => {
+      body += chunk.toString();
+    });
+
+    res.on('end', () => {
+      if (!errored) {
+        cb(null, { statusCode }, body);
+      }
+    });
+  });
+
+  req.on('error', (e) => {
+    errored = true;
+    cb(e);
+  });
+
+  // Write data to request body
+  req.write(data);
```

Comment:
Previously, the request was POST-ing as a urlencoded form, but here, you're just posting the JSON data raw without any prefix.

So, eg, if the object was `{foo: 'bar'}`, then previously, the post body would've been `json=%7B%22foo%22%3A%22bar%22%7D`, but now it's `{"foo":"bar"}`.

That's almost certainly why it's failing.

Try something like this:

```js
req.write(`json=${encodeURIComponent(data)}`)
```